### PR TITLE
New version: Mumble Server version 1.5.634

### PR DESCRIPTION
**Pull request by Kissaki**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Note: Version 1.4 published x86 and x64 releases. Version 1.5 only publishes x64.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154396)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\Mumble\server\mumble-server.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 881.)

File: manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-05-19
+Installers:
```
Comment (stephengillie) [inline manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.installer.yaml:15]:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with: 
image

File: manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Architecture: x64
+  InstallerUrl: https://github.com/mumble-voip/mumble/releases/download/v1.5.634/mumble_server-1.5.634.x64.msi
+  InstallerSha256: 69e4a9737043d785b6b78144ecaa348a5817282e4b4cd056db0a465818e6b766
+  ProductCode: '{B6DABF0E-FB65-4B2E-B6B4-9B7207C10F15}'
+ManifestType: installer
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.installer.yaml:21]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- voice-chat
+- voip
+# Agreements: 
+# ReleaseNotes: 
+ReleaseNotesUrl: https://github.com/mumble-voip/mumble/releases/tag/v1.5.634
+# PurchaseUrl: 
+# InstallationNotes: 
+# Documentations: 
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.locale.en-US.yaml:33]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: Mumble.Mumble.Server
+PackageVersion: 1.5.634
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.yaml:5]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
I forgot to mention earlier - This manifest version has been depreciated, but most fields are identical to the newest version. In most cases, just increasing the ManifestVersion is enough to pass this check.

File: manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.installer.yaml (hunk 1 above)
Comment (Kissaki) [reply manifests/m/Mumble/Mumble/Server/1.5.634/Mumble.Mumble.Server.installer.yaml:18]: Install in Sandbox via `powershell .\Tools\SandboxTest.ps1 manifests\m\Mumble\Mumble\Server\1.5.634` installs and starts up/runs fine for me.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**